packages/react-native-reanimated/apple/reanimated/apple/keyboardObserver/REAKeyboardEventObserver.mm

```diff
@@ -323,11 +350,16 @@ - (void)updateKeyboardHeightDuringInteractiveDismiss:(CGPoint)oldKeyboardFrame
   }
   float windowHeight = keyboardView.window.bounds.size.height;
   float keyboardHeight = keyboardView.frame.size.height;
+  if (keyboardHeight == 0 || keyboardView.frame.size.width != keyboardView.window.bounds.size.width) {
+    return;
+  }
```

Comment:
I'd love some comment why this is needed too. Some person reading it in a 2 years time might not get that the width check is for floating keyboard.